src/main/java/funda/prog101/Boiler.java

```diff
@@ -0,0 +1,5 @@
+package main.java.funda.prog101;
+
+public class Boiler {
```

Comment:
In the diagram Class it was defined like a extends of Container, it would be public class Boiler extends Contatiner {